src/components/tabContainer/Tabs.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Tab from './Tab';
+
+class Tabs extends React.Component {
+  state = {
+    tabs: [...this.props.tab],
+    activeTab: 0,
+  }
+
+  selected = (selected) => {
+    this.setState((state) => {
+      const newActive = state.activeTab = selected.id;
+
+      return { activeTabs: newActive };
+    });
+  }
```

Comment:
я не можу з цим прийняти, бо так не робиться.
якщо тобі в selected приходить якес id, то його і записуй
```suggestion
  setActiveTab = (selected) => this.setState((state) => ({
      activeTab: selected.id,
    }));
```
Вірю що Зоракс може дуже класний пацан, але то не годиться)